jupyter_alabaster_theme/static/pcss/navbar.pcss

```diff
@@ -62,3 +62,10 @@
     }
   }
 }
+
+@media (max-width: 768px) {
```

Comment:
Can you put the `@media query` in mediaqueries.pcss ? It's much easier if all media queries are in one place